### PR TITLE
[thread-cert] add case 9.2.3 ActiveDatasetGet using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_9_2_03_ActiveDatasetGet.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+        self.nodes[LEADER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[COMMISSIONER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[COMMISSIONER].get_state(), 'router')
+        self.simulator.get_messages_sent_by(LEADER)
+
+        self.collect_leader_aloc(LEADER)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_03_ActiveDatasetGet.py:88]: nit: Leader ALOC is fixed

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 
with nits

File: tests/scripts/thread-cert/node.py [hunk 2]
```diff
@@ -1501,48 +1501,20 @@ def send_mgmt_active_set(
...
-        self,
-        active_timestamp=None,
-        channel=None,
-        channel_mask=None,
-        extended_panid=None,
-        panid=None,
-        master_key=None,
-        mesh_local=None,
-        network_name=None,
-        binary=None,
```
[the remark this answers is not in the document]
Comment (FiveDimensions) [reply tests/scripts/thread-cert/node.py:1514]: only used in 9.2.4 and 9.2.5 with no args, which has been considered in current modification

[the remark this answers is not in the document]
Comment (simonlingoogle) [reply tests/scripts/thread-cert/node.py:1514]: fine to remove them

**Review by jwhui — APPROVED** [review]

**Pull request merged**